pyomo/contrib/incidence_analysis/util.py

```diff
@@ -91,6 +95,7 @@ def solve_strongly_connected_components(
         solver=None,
         solve_kwds=None,
         calc_var_kwds=None,
+        tee=False
```

Comment:
The existing `solve_kwds` is intended to pass arguments like this to the solver's `solve` method. I'm fine with a dedicated `tee` optional argument, but can you add a check that `tee` and `solve_kwds` don't contain conflicting instructions about whether to "tee" the solver's output?

Reply:
Yes, it would conflict it this case. I had misinterpreted this as being the solver options argument.

Reply:
Yep, I see the potential confusion. I intended `solver` to already be loaded with options. FWIW, I intend to write a registered "Pyomo solver", e.g. `SolverFactory("incidence.scc")`, which can handle `tee` and options in a more consistent manner.

Reply:
I removed the additional 'tee' argument.